sentences/de-CH/_common.yaml

```diff
@@ -45,17 +49,17 @@ lists:
       - in: "(f|fahrenheit)"
         out: "Fahrenheit"
 expansion_rules:
-  name: "([( de | der | die | ds | s )] {name} | [( dr | ds | s | d )] {name})"
-  area: "[( idr | im | ir | ide | idä | i dä | i de )] {area}"
-  a_y: "( a | y | ih )"
-  ab_us: "( ab | us | uus | uis )"
-  what_is: "( was isch | wasch )"
+  name: "[(d'|d|de|dr|der|die|ds|s)] {name}"
```

Comment:
Here is my opinion. As I for sure cannot speak Swiss :)
I assume sentences are: d'NAME and d NAME, de NAME, dr NAME... if so, I would prefer to change rule to: 
`"[(d'|d |de |dr |der |die |ds |s )]{name}"`

but it is a variant, not something thaht is critical.